libraries/AP_Devo_Telem/AP_Devo_Telem.cpp

```diff
@@ -48,12 +48,12 @@ void AP_DEVO_Telem::init()
 }
 
 
-uint32_t AP_DEVO_Telem::gpsDdToDmsFormat(float ddm)
+uint32_t AP_DEVO_Telem::gpsDdToDmsFormat(int32_t ddm)
 {
-    int32_t deg = (int32_t)ddm;
-    float mm = (ddm - deg) * 60.0f;
+    int32_t deg = (int32_t)(ddm / 1e7);
+    double mm = (ddm / 1.0e7 - deg) * 60.0f;
```

Comment:
```suggestion
    int32_t deg = (int32_t)(ddm * 1e-7);
    float mm = (ddm * 1.0e-7 - deg) * 60.0f;
```

Multiplications are *much* cheaper than divisions.  That's important in the embedded sphere.

Reply:
Done!